frontend/src/scenes/insights/LineGraph/LineGraph.tsx

```diff
@@ -310,40 +314,27 @@ export function LineGraph(props: LineGraphProps): JSX.Element {
                             const referenceDataPoint = tooltip.dataPoints[0] // Use this point as reference to get the date
                             const dataset = datasets[referenceDataPoint.datasetIndex]
 
-                            const altTitle =
-                                tooltip.title && (dataset.compare || tooltipPreferAltTitle) ? tooltip.title[0] : '' // When comparing we show the whole range for clarity; when on stickiness we show the relative timeframe (e.g. `5 days`)
-                            const referenceDate = !dataset.compare
-                                ? dataset.days?.[referenceDataPoint.dataIndex]
-                                : undefined
-                            const seriesData = tooltip.body
-                                .flatMap(({ lines }) => lines)
-                                .map((component, idx) => ({
-                                    id: idx,
-                                    component,
-                                }))
-
                             ReactDOM.render(
                                 <Provider store={getContext().store}>
-                                    {featureFlags[FEATURE_FLAGS.NEW_INSIGHT_TOOLTIPS] ? (
-                                        <InsightTooltip
-                                            referenceDate={referenceDate}
-                                            altTitle={altTitle}
-                                            seriesData={seriesData}
-                                            useAltTitle={tooltipPreferAltTitle}
-                                            hideHeader={isHorizontal}
-                                            hideInspectActorsSection={!(onClick && showPersonsModal)}
-                                        />
-                                    ) : (
-                                        <LEGACY_InsightTooltip
-                                            referenceDate={referenceDate}
-                                            altTitle={altTitle}
-                                            interval={interval}
-                                            bodyLines={seriesData}
-                                            preferAltTitle={tooltipPreferAltTitle}
-                                            hideHeader={isHorizontal}
-                                            inspectPersonsLabel={onClick && showPersonsModal}
-                                        />
-                                    )}
+                                    <InsightTooltip
```

Comment:
On a related (but loosely) note, there's still a problem where navigating away from the scene while the insight tooltip is shown results in the insight tooltip being totally stuck:
<img width="284" alt="Screen Shot 2022-01-06 at 12 04 28" src="https://user-images.githubusercontent.com/4550621/148375209-5921d1e3-e038-4852-9e04-240babfe54f1.png">
We should somehow stop rendering it once the `LineGraph` component is not shown anymore.

Reply:
This should be fixed now